1.9/usage/cli/command-reference/dcos-config/dcos-config-show.md

```diff
@@ -0,0 +1,32 @@
+---
+post_title: dcos config show
+menu_order: 2
+--- 
+
+# Description
+This command prints the DC/OS configuration file contents.
+
+# Usage
+
+```bash
+dcos config show [<name>]
+``` 
+
+# Options
+
+| Name, shorthand | Default | Description |
+|---------|-------------|-------------|
+| `--help, h`   |             |  Print usage. |
+| `--info`   |             |  Print a short description of this subcommand. |
+| `--version`   |             |  Print version information. |
+| `<name>`   |             |  The name of the property. |
+
+# Parent command
+
+| Command | Description |
+|---------|-------------|
+| [dcos config](/docs/1.9/usage/cli/command-reference/dcos-config/) |  Manage DC/OS configuration. |
+
+# Examples
```

Comment:
To see a specific config value, for example: core.dcos_url, run: `dcos config show core.dcos_url`
To see all config values set run `dcos config show`